cds_books/migrator/cli.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import absolute_import, print_function
+
+import json
+import os
+import re
+
+import click
+from flask import current_app
+from flask.cli import with_appcontext
+from invenio_app_ils.records.api import Series, Keyword
+from invenio_db import db
+from invenio_migrator.cli import _loadrecord, dumps
+from invenio_pidstore.errors import PIDAlreadyExists
+from invenio_pidstore.models import PersistentIdentifier
+from invenio_records import Record
+
+from cds_books.migrator.records import CDSParentRecordDumpLoader
+
+
+def load_dumps_from_dir(source_dir, rectype):
+    with click.progressbar(os.listdir(source_dir)) as bar:
+        for filename in bar:
+            click.echo('Loading dump {0}'.format(filename))
+            if re.match(r'^'+rectype+'_(\d+)(_\d+)*.json$', filename):
```

Comment:
Now we changed the migrator to store all dumps in a single file so this will break. Should we change it back to store each dump in its own file or keep it as-is in a single file?